lib/ansible/plugins/action/net_template.py

```diff
@@ -93,6 +93,16 @@ def _handle_template(self):
         except IOError:
             return dict(failed=True, msg='unable to load src file')
 
+        # Create a template search path in the following order:
+        # [working_path, _role_path, *_block._role_paths, dirname(source)]
+        searchpath = [working_path]
+        if self._task._role is not None:
+            searchpath.append(self._task._role._role_path)
+            if self._task._block._dep_chain:
+                for role in sorted(self._task._block.get_dep_chain()):
```

Comment:
What is the purpose to sorting this? Forgive me if I'm being obtuse, I'm just trying to understand what is happening here, and this step is a little unclear.